pkg/kubelet/volume/reconciler/reconciler.go

```
@@ -0,0 +1,231 @@
+							volumeToMount.VolumeName,
+							volumeToMount.VolumeSpec.Name(),
+							volumeToMount.PodName,
+							volumeToMount.Pod.UID,
+							rc.controllerAttachDetachEnabled,
+							addVolumeNodeErr)
+					}
+				} else {
+					// Volume is not attached to node, kubelet attach is enabled, volume implements an attacher,
+					// so attach it
+					volumeToAttach := operationexecutor.VolumeToAttach{
+						VolumeName: volumeToMount.VolumeName,
+						VolumeSpec: volumeToMount.VolumeSpec,
+						NodeName:   rc.hostName,
+					}
+					glog.V(12).Infof("Attempting to start AttachVolume for volume %q (spec.Name: %q)  pod %q (UID: %q)",
+						volumeToMount.VolumeName,
+						volumeToMount.VolumeSpec.Name(),
+						volumeToMount.PodName,
+						volumeToMount.Pod.UID)
+					err := rc.operationExecutor.AttachVolume(volumeToAttach, rc.actualStateOfWorld)
+					if err == nil {
+						glog.Infof("AttachVolume operation started for volume %q (spec.Name: %q) pod %q (UID: %q)",
+							volumeToMount.VolumeName,
+							volumeToMount.VolumeSpec.Name(),
+							volumeToMount.PodName,
+							volumeToMount.Pod.UID)
+					}
+				}
+			} else if !volMounted || cache.IsRemountRequiredError(err) {
+				// Volume is not mounted, or is already mounted, but requires remounting
+				remountingLogStr := ""
+				if cache.IsRemountRequiredError(err) {
+					remountingLogStr = "Volume is already mounted to pod, but remount was requested."
+				}
+				glog.V(12).Infof("Attempting to start MountVolume for volume %q (spec.Name: %q) to pod %q (UID: %q). %s",
+					volumeToMount.VolumeName,
+					volumeToMount.VolumeSpec.Name(),
+					volumeToMount.PodName,
+					volumeToMount.Pod.UID,
+					remountingLogStr)
+				err := rc.operationExecutor.MountVolume(
+					rc.waitForAttachTimeout,
+					volumeToMount.VolumeToMount,
+					rc.actualStateOfWorld)
+				if err == nil {
```

swallowing error here